src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -134,49 +91,71 @@ def test_WINNF_FT_S_FAD_1(self):
-                    for record in data['recordData']:
-                        # Verify the response files with CbsdData.schema.json Object schema
-                        self.assertContainsRequiredFields("CbsdData.schema.json", record)
-                        self.assertEqual(1, len(record['grants']))
-                        if record['registration']['fccId'] == device_a['fccId']:
-                            self.assertCbsdRecordEqual(record['registration'], device_a)
-                            self.assertGrantRecord(record['grants'][0], grant_a, grant_response[0])                   
-                        else:
-                            self.assertEqual(record['registration']['fccId'], device_c['fccId'])
-                            self.assertCbsdRecordEqual(record['registration'], device_c)
-                            self.assertGrantRecord(record['grants'][0], grant_c, grant_response[1])
-                if activity_dump_file['recordType'] == 'esc_sensor':
-                    self.assertEqual(1, len(data['recordData']))
-                    # Verify the response file of Esc Sensor
-                    self.assertContainsRequiredFields("EscSensorRecord.schema.json", data['recordData'][0])
-                    self.assertDictEqual(esc_sensor_record, data['recordData'][0])
-                if activity_dump_file['recordType'] == 'zone':
-                    self.assertEqual(1, len(data['recordData']))
-                    # Verify the response file of PPA
-                    self.assertContainsRequiredFields("zoneData.schema.json", data['recordData'][0])
-                    self.assertEqual(ppa_record['id'], data['recordData'][0]['id'])
-                    self.assertEqual(ppa_record['usage'], data['recordData'][0]['usage'])
-                    self.assertEqual(ppa_record['terminated'], data['recordData'][0]['terminated'])
-                    self.assertEqual(ppa_record['zone']['features'][0]['geometry']['coordinates'],\
-                                      data['recordData'][0]['zone']['features'][0]['geometry']['coordinates'])
-                    self.assertEqual(set(ppa_record['ppaInfo']['palId']),\
-                                      set(data['recordData'][0]['ppaInfo']['palId']))
-                    self.assertEqual(set(ppa_record['ppaInfo']['cbsdReferenceId']), \
-                                     set(data['recordData'][0]['ppaInfo']['cbsdReferenceId']))
-                    self.assertEqual(ppa_record['ppaInfo']['ppaBeginDate'],\
-                                      data['recordData'][0]['ppaInfo']['ppaBeginDate'])
-                    self.assertEqual(ppa_record['ppaInfo']['ppaExpirationDate'],\
-                                      data['recordData'][0]['ppaInfo']['ppaExpirationDate'])                                    
\ No newline at end of file
+        for activity_dump_file in response['files']:
```

Comment:
If a SAS responds with no files, will it pass or fail this test? What will happen if it includes only the CBSD file? What will happen if it includes two CBSD files, one with each device (which is technically allowed)? What will happen if it includes two CBSD files, each with both devices inside?

Reply:
Please reconsider this portion of the comment:
> What will happen if it includes two CBSD files, one with each device (which is technically allowed)? What will happen if it includes two CBSD files, each with both devices inside?

Reply:
I think the test now is better for this case, could you check it?
thank you :)